components/chainhook-sdk/src/indexer/tests/helpers/stacks_shapes.rs

```diff
@@ -3739,3 +3739,91 @@ pub fn get_vector_052() -> Vec<(BlockEvent, StacksChainEventExpectation)> {
         ),
     ]
 }
+
+/// Vector 053: Generate the following blocks
+///
+///  A1(0) - B1(1) - C1(3)
```

Comment:
> A1(0)
I've added the `(0)` to mean that this data already existed on startup (so we're seeding the block pool with this block)